### PR TITLE
Feat/panic

### PR DESCRIPTION
**Pull request by xl3ehindTim**
Change the panic feature to the new queue structure

### CODE REVIEW

File: src/router/deleteImageRouter.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ router.post('/', async (req, res) => {
   }
   try {
     await removeLatestPromptModel();
+	await addApprovedPromptModel('vulcano');
```
Comment (BartvanEijkelenburg) [inline src/router/deleteImageRouter.ts:13]:
```suggestion
	await addApprovedPromptModel('vulcano');
```

**Review by BartvanEijkelenburg — CHANGES_REQUESTED** [review]
Paar dingetjes besproken f2f

**Comment by xl3ehindTim** [comment]
More panic prompts should be added

File: src/model/promptModel.ts [hunk 2]
```diff
@@ -14,6 +15,7 @@ const promptModel = new Schema<IPrompt>(
     prompt: { type: String, required: true },
     approved: { type: Boolean, default: false, required: true },
     isUsed: { type: Boolean, default: false, required: true },
+    hasPriority: { type: Boolean, default: false, required: false },
```
Comment (BartvanEijkelenburg) [inline src/model/promptModel.ts:18]:
```suggestion
    isPanic: { type: Boolean, default: false, required: false },
```

File: src/model/promptModel.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ export interface IPrompt extends Document {
   prompt: string;
   approved: boolean;
   isUsed: boolean;
+  hasPriority: boolean;
```
Comment (BartvanEijkelenburg) [inline src/model/promptModel.ts:8]:
```suggestion
  isPanic: boolean;
```

**Review by BartvanEijkelenburg — COMMENTED** [review]
I think this is a nice solution :) 
Well done!

**Review by BartvanEijkelenburg — APPROVED** [review]
LGTM

**Pull request merged**